docs/how-to-guides/setting-up-social-login.md

```diff
@@ -0,0 +1,25 @@
+# Setting Up Social Login Integration
+
+Streamline user authentication by enabling social login for GitHub and Drupal.org. This enhances usability while maintaining security.
```

Comment:
Why is this a How-to? It's already set up for contrib tracker. Why would they want to do it again?